docs/topics/cache.txt

```diff
@@ -548,6 +548,15 @@ flag on the connection's socket::
         }
     }
 
+``pymemcache`` by default sets the following options (you can override them on
+your :setting:`OPTIONS <CACHES-OPTIONS>`)::
+
+    'OPTIONS': {
+        'allow_unicode_keys': True,
+        'default_noreply': False,
+        'serde': pymemcache.serde.pickle_serde,
+    }
```

Comment:
I would move this to the ["Memcached" section](https://docs.djangoproject.com/en/dev/topics/cache/#memcached), just before _"A final point about...."_:
```
By default, the ``PyMemcacheCache`` backend sets the following options (you can
override them in your :setting:`OPTIONS <CACHES-OPTIONS>`)::

    'OPTIONS': {
        'allow_unicode_keys': True,
        'default_noreply': False,
        'serde': pymemcache.serde.pickle_serde,
    }

A final point about...
```

Reply:
Yes, you're right, it fits better there.